clients/banking/src/locales/en.json

```diff
@@ -724,10 +725,78 @@
   "transfer.new.sendFullBalance.description": "Choose how much to leave in your account, and transfer the rest to your beneficiary",
   "transfer.new.sendRegularTransfer": "Send a regular standing order instead",
   "transfer.new.sendRegularTransfer.description": "You will send a regular standing order",
+
```

Comment:
I think the lines will be removed when pushing / pulling translations to Localazy